tests/integration-tests/conftest.py

```diff
@@ -1354,7 +1354,7 @@ def to_pascal_case(snake_case_word):
 
 
 @pytest.fixture()
-def run_benchmarks(request, mpi_variants, test_datadir, instance, os, region, benchmarks):
+def run_benchmarks(request, mpi_variants, test_datadir, instance, os, region, benchmarks):  # noqa: C901
```

Comment:
The C901 is an alert to suggest us to try to refactor the code into a subfunctions, because probably we have too many if/else branches or loops